packages/api-gateway/src/drivers/http/routes/auth/schema.js

```diff
@@ -0,0 +1,20 @@
+/** @type {import('fastify').RouteOptions['schema']} */
```

Comment:
This is going to be a health check? or it's just a temporary schema and endpoint?

Reply:
is only a schema and temporary endpoint. in the new issue I am still working on, I have already removed the "auth/*" path.